q2_diversity/_ordination.py

```diff
@@ -28,3 +30,80 @@ def pcoa(distance_matrix: skbio.DistanceMatrix,
 def pcoa_biplot(pcoa: skbio.OrdinationResults,
                 features: pd.DataFrame) -> skbio.OrdinationResults:
     return skbio.stats.ordination.pcoa_biplot(pcoa, features)
+
+
+def tsne(distance_matrix: skbio.DistanceMatrix,
+         **kwargs) -> skbio.OrdinationResults:
```

Comment:
```suggestion
def tsne(distance_matrix: skbio.DistanceMatrix) -> skbio.OrdinationResults:
```

`**kwargs` is invalid when defining a QIIME 2 Action - you need specify a parameter for every argument. This is so that the QIIME 2 framework can build the interface that matches the signature.